lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -215,12 +215,16 @@ int main(int argc, char* argv[]) {
       MAX_URL_LENGTH);
   initialize_sentry(SENTRY_TAG_SESSIOND, &sentry_config);
 
-  bool converged_access          = false;
+  bool enable5g_features         = false;
   uint32_t session_max_rtx_count = 0;
   // Check converged sessiond is enabled or not
-  if ((config["converged_access"].IsDefined()) &&
-      (config["converged_access"].as<bool>())) {
-    converged_access = true;
+  if ((config["enable5g_features"].IsDefined()) &&
+      (config["enable5g_features"].as<bool>())) {
+    enable5g_features = true;
+    MLOG(MINFO) << "Setting enable5g_feature flag from sesiond.yml";
```

Comment:
can you use a clearer message: `Enabling enable5g_feature`

Reply:
Yeah Updated the Log message as you suggested.